src/en/draft-benchmarking.md

```diff
@@ -0,0 +1,147 @@
+# Why benchmarking
+
```

Comment:
It could probably do with a bit more explanation here - what are the benefits of adding a benchmark to your charm (testing against different architectures/ cloud providers, identifying bottlenecks and optimum config values  or whatever). In fairness, it should probably make it obvious that benchmarks are a 'nice-to-have' optional feature.